phpstan.neon

```diff
@@ -0,0 +1,10 @@
+parameters:
+    level: 0
```

Comment:
I'm assuming we are going to start at `level 0` here and work our way up?  Is there a big difference in running different levels here?

Reply:
We can start at 4 since my PR brought us up to that IIRC.

Reply:
Moved to 4